homeassistant/components/abode/__init__.py

```diff
@@ -83,6 +84,12 @@ class AbodeSystem:
     logout_listener: CALLBACK_TYPE | None = None
 
 
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Abode component."""
+    setup_hass_services(hass)
```

Comment:
When moving the service setup we need to make sure that the service handler handles a missing or not loaded config entry and raise appropriate exceptions in those cases. That's not done here.